.github/workflows/subgraph-documentation-update.yml

```diff
@@ -15,7 +15,7 @@ on:
       - '.github/workflows/subgraph-deploy.yaml'
 
 jobs:
-  update_docs:
+  update_subgraph_docs:
```

Comment:
I'm not sure this is the problem taking into account they are on different files